### PR TITLE
[BI-1692] - Change role names

### PR DESCRIPTION
**Pull request by HMS17**
# Description
**Story:** [BI-1692 - Change role names](https://breedinginsight.atlassian.net/browse/BI-1692)
TAF changes in response to changing role domains as follows:
- admin -> System Administrator
- breeder -> Program Administrator
- member -> Read Only

Areas updated to use new role domains:
- README
- populate-taf-data.sql
- Features
- Step definitions

# Dependencies
bi-web: [feature/BI-1692](https://github.com/Breeding-Insight/bi-web/pull/388)
bi-api: [feature/BI-1692](https://github.com/Breeding-Insight/bi-api/pull/377)

# Testing
[_Please include a link to a successful run of TAF for this change_](https://github.com/Breeding-Insight/taf/actions/runs/10199847184)

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have tested my code and ensured it meets the acceptance criteria of the story
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to documentation

### CODE REVIEW

**Review by cvacalares — APPROVED** [review]
Looks good.

File: src/features/LoginByRoleMemberTests.feature [hunk 1]
```diff
@@ -15,7 +15,7 @@ Feature: Logging with Member
 		And user can see "Program Administration" in navigation
 
 	@BI-823
-	Scenario: Logging in as a breeder of one program
+	Scenario: Logging in as a Read Only of one program
```
Comment (nickpalladino) [inline src/features/LoginByRoleMemberTests.feature:18]: Was this an issue in the description of the test?

File: src/features/LoginByRoleMemberTests.feature [hunk 2]
```diff
@@ -28,14 +28,14 @@ Feature: Logging with Member
 		And user can see "Program Administration" in navigation
 
 	@BI-845
-	Scenario: Logging in as a breeder of one program
+	Scenario: Logging in as a Read Only of one program
```
Comment (nickpalladino) [inline src/features/LoginByRoleMemberTests.feature:31]: Seems like this was just a labeling issue of the scenario like the last one

Comment (HMS17) [reply src/features/LoginByRoleMemberTests.feature:31]: Yes, I figured I might as well fix the labels since I was changing them anyway.

**Review by nickpalladino — APPROVED** [review]

**Review by mlm483 — APPROVED** [review]

**Pull request merged**